packages/cardano-services/test/StakePoolSeach/StakePoolSearchHttpService.test.ts

```diff
@@ -468,6 +483,75 @@ describe('StakePoolSearchHttpService', () => {
           });
         });
       });
+
+      describe('stake pools sort', () => {
+        const filterArgs: StakePoolQueryOptions = {
+          filters: {
+            identifier: {
+              _condition: 'or',
+              values: [
+                { ticker: 'TEST' },
+                { name: STAKE_POOL_NAME },
+                { id: Cardano.PoolId('pool1jcwn98a6rqr7a7yakanm5sz6asx9gfjsr343mus0tsye23wmg70') }
+              ]
+            }
+          }
+        };
+
+        it('sort by name desc order', async () => {
+          const response = await doServerRequest(setSortCondition({}, 'desc', 'name'));
+          expect(response).toMatchSnapshot();
+        });
+
+        it('sort by name asc order', async () => {
+          const response = await doServerRequest(setSortCondition({}, 'asc', 'name'));
+          expect(response).toMatchSnapshot();
+        });
+
+        it('if sort not provided defaults by name asc order', async () => {
+          const response = await doServerRequest({});
+          expect(response).toMatchSnapshot();
```

Comment:
Maybe a better test here is to query explicitly and compare the result, rather than writing a snapshot that needs to be verified

Reply:
Addressed [79ff5bc](https://github.com/input-output-hk/cardano-js-sdk/pull/252/commits/79ff5bcc52709801b3afd7d2ddb9b65950738d58)